src/VolleyManagement.API/Controllers/AccountController.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace VolleyManagement.API.Controllers
+{
+    using Google.Apis.Auth;
+    using Microsoft.AspNet.Identity;
+    using Microsoft.AspNetCore.Authentication;
+    using Microsoft.AspNetCore.Authentication.Cookies;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Newtonsoft.Json;
+    using System.Collections.Generic;
+    using System.Security.Claims;
+    using System.Threading.Tasks;
+    using Model;
+    using Contracts;
+    using Contracts.Authentication;
+    using Contracts.Authentication.Models;
+    using Crosscutting.Contracts.Providers;
+
+    [Produces("application/json")]
+    [Route("api/[controller]/[action]")]
+    public class AccountController : Controller
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 23 to the maximum authorized 20 or less. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1200)

Reply:
Suppress this one